.scripts/prepublish.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+const shell = require('shelljs');
+const chalk = require('chalk');
```

Comment:
Hi! We already use chalk in `@kadira/storybook`, so I added it as devDependncies as well. Yes it works fine in windows and such output looks more readable